### PR TITLE
Adding WIP webforms and Maestro templates.

### PR DESCRIPTION
**Pull request by madsnorgaard**

### CODE REVIEW

**Review by agger-magenta — APPROVED** [review]
Ja, lad os endelig få de eksempler ind!

File: src/Plugin/EngineTasks/MaestroSelectContentTask.php [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getTaskEditForm(array $task, $templateMachineName) {
+        $form = array(
+            '#markup' => t('Select a Content Item.'),
+        );
+
+        $content_type_options = array(""=>"All Content Types");
```
Comment (lats1) [inline src/Plugin/EngineTasks/MaestroSelectContentTask.php:87]: I guess an empty string as key in an array will work, but why not use "all" or something similar for better understanding? And also, should "All Content Types" be translated?

**Review by lats1 — CHANGES_REQUESTED** [review]

File: os2forms_forloeb.module [hunk 2]
```diff
@@ -28,3 +28,84 @@ function os2forms_forloeb_install() {
...
+//    '#id' => 'approve_content',
+//    '#type' => 'radios',
+//    '#options' => array(
+//      0 => t("Assign for Review"),
+//      1 => t("Approve")
+//    ),
+//    '#default_value' => 0,
+//    '#title' =>  t('Review or Approve Content '),
+//    '#required' => true,
+//  );
```
[the remark this answers is not in the document]
Comment (madsnorgaard) [reply os2forms_forloeb.module]: This code has been removed as it is not necessary for the review workflow.

File: src/Plugin/EngineTasks/MaestroSelectContentTask.php (hunk 1 above)
Comment (madsnorgaard) [reply src/Plugin/EngineTasks/MaestroSelectContentTask.php]: Changes committed per comment. All good input.

**Review by lats1 — APPROVED** [review]

**Pull request merged**